### PR TITLE
(#2828) updateFacility and addFacility returns modified facility, upd…

### LINKED ISSUE #2828: [BUG] Newly added facility doesn't show up without refreshing the page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Newly added facility doesn't show up without refreshing the page

### Steps To Reproduce

- Add a facility
- Observe that the new facility doesn't show up when you save

### Expected Behavior

Newly added facility should show up without refreshing the page

### Additional context

https://user-images.githubusercontent.com/4952042/138180648-ff674c71-69b9-4c2b-8572-81590611f78e.mp4

### Does this issue have a priority label?

- [X] I have added a priority label

### Has issue been assigned to a team?

- [X] I have added a team label

### PR DESCRIPTION
**Pull request by alismx**
## Related Issue or Background Info

- Resolves #2828

## Changes Proposed

- Add or Update facilities in the store after saving.
- Do not cache Facilities on the manage facilities page
- Backwards compatible

### UI
- [ ] Any changes to the UI/UX are approved by design 
- [ ] Any new or updated content (e.g. error messages) are approved by design 

### Testing
- [ ] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [ ] Database changes are submitted as a separate PR
  - [ ] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [ ] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [ ] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
  - [ ] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [ ] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [ ] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [ ] Any dependencies introduced have been vetted and discussed

## Cloud
- [ ] DevOps team has been notified if PR requires ops support
- [ ] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

**Comment by alismx** [comment]
Synk had a rate limiting failure. It doesn't appear to be due to the changes in this PR.

**Review by zdeveloper — APPROVED** [review]
LGTM!

File: backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java [hunk 1]
```diff
@@ -159,14 +102,12 @@ public ApiFacility addFacilityNew(
     return new ApiFacility(created);
   }
 
-  public ApiFacility updateFacility(
-      UUID facilityId,
+  public ApiFacility addFacilityNew(
```
Comment (emmastephenson) [inline backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java:105]: nit: can you add a comment clarifying the difference between `addFacility` and `addFacilityNew`?

**Review by emmastephenson — DISMISSED** [review]
LGTM!

**Review by nathancrtr — APPROVED** [review]
LGTM, thanks for the very thorough work on this!

**Review by emmastephenson — DISMISSED** [review]

**Review by nathancrtr — DISMISSED** [review]
Re-approving after additional commits.

File: backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java (hunk 1 above)
Comment (zdeveloper) [reply backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java:110]: great callout by emma, can you add the same comments for `updateFacility` and `updateFacilityNew`. and also add a ticket in the backlog to implement this change so we dont forget about it.

**Review by zdeveloper — APPROVED** [review]

**Comment by alismx** [comment]
@emmastephenson @zdeveloper I've added the comment to updateFacility and updateFacilityNew sections and created a ticket to go back and pull out addFacilityNew and updateFacilityNew. https://github.com/CDCgov/prime-simplereport/issues/3221

**Review by emmastephenson — APPROVED** [review]

**Review by nathancrtr — APPROVED** [review]
Re-approving

**Pull request merged**